testing/docker/rabbitmq_setup.go

```diff
@@ -21,7 +21,7 @@ var (
 )
 
 // SetupRabbitMQ sets up a RabbitMQ broker
-func SetupRabbitMQ(testName string) (string, error) {
+func SetupRabbitMQ(testName string, timeOut time.Duration) (string, error) {
```

Comment:
I would have set it up as an optional parameter ( ..., optional ...int)
...
and then check if [0] has a value and use it otherwise use a longer than 30 seconds default..